contracts/BattleShipGame.vy

```diff
@@ -0,0 +1,137 @@
+# Define the board size
```

Comment:
vyper version needs to be the first lines of code, otherwise ape test won't run
# @version ^0.3.10
# Data structuress